modules/oe_content_publication/oe_content_publication.info.yml

```diff
@@ -6,6 +6,8 @@ type: module
 core: 8.x
 dependencies:
   - oe_content:oe_content
+  - oe_content:oe_content_entity_contact
+  - oe_content:oe_content_organisation_reference
   - oe_content:oe_content_documents_field
```

Comment:
Add these two deps as new configuration requires them. Also enable them in an update path.
```
inline_entity_form
path
```